### PR TITLE
Clarify "community staff" in CoC

### PR DESCRIPTION
**Pull request by ceedubs**
This is a first pass at a resolution to #33.

I've added email links for a few people that already had their email
addresses publicly available and that I felt confident would be happy to
be on this contact list. We can definitely add more contacts if there
are others who would be willing to be contacts.

### CODE REVIEW

**Comment by milessabin** [comment]
:+1:

**Comment by non** [comment]
:+1:

**Comment by tpolecat** [comment]
:+1:

**Comment by adelbertc** [comment]
:+1:

**Comment by ceedubs** [comment]
@kpfell as you were so helpful with #53, do you have any input on this as well?

**Comment by larsrh** [comment]
Just want to add that there is also the general @typelevel/owners address posted on the [about page](http://typelevel.org/about.html), although it might be better to list personalized addresses.

**Comment by stew** [comment]
:+1:

**Comment by kpfell** [comment]
@ceedubs I'd rephrase this slightly to address the reader, as a form of encouragement:

> If you have any questions or concerns, please contact a member of the community staff, such as [...]

Since you have the email address that @larsrh mentioned, as well, I would definitely tack a note onto the end about it--though, if a group of people has access to that account, I would mention who those people are.

**Comment by ceedubs** [comment]
A concern that I have here is that documentation is prone to going out of date. We could list the current people that get `info@typelevel.org` emails, but that could change in the future, and if we forget to update the Code of Conduct accordingly, I would hate for an email to go to someone that it wasn't intended to go to. I could leave a note that the members of this email address could change over time, but that might not be that helpful to someone who wants to be sure about who will see their email.

What do you think?

**Comment by ceedubs** [comment]
By the way, @kpfell, I do like like the suggestion of addressing the reader for encouragement and will definitely incorporate that change regardless of what's decided about the Typelevel email address.

**Comment by non** [comment]
Let's just list people explicitly. In addition to avoiding the out-of-date documentation problem, it also makes it easy for someone to avoid including someone they don't want to.

For example, if someone wanted to report me for a CoC violation it would be bad if they sent it to info@ without realizing that I would be included.

**Comment by non** [comment]
:+1:

**Comment by kpfell** [comment]
@ceedubs 

> We could list the current people that get info@typelevel.org emails, but that could change in the future, and if we forget to update the Code of Conduct accordingly, I would hate for an email to go to someone that it wasn't intended to go to. I could leave a note that the members of this email address could change over time, but that might not be that helpful to someone who wants to be sure about who will see their email.

Ah, okay--fair enough! I wasn't sure if you had a GitHub team set up that corresponded with people able to access the account.

:+1: if not. :wink:

**Comment by milessabin** [comment]
:+1:

**Comment by ceedubs** [comment]
@kpfell I'm not sure exactly how the `info@typelevel.org` email address is set up. Paging @larsrh.

**Comment by larsrh** [comment]
@ceedubs It's an alias on my personal mail server.

**Comment by ceedubs** [comment]
@larsrh okay thanks. I think we can stick with what we have for now then.

**Comment by ceedubs** [comment]
I think this is good to merge.
